db/schema.rb

```diff
@@ -36,6 +50,32 @@
     t.index ["key"], name: "index_active_storage_blobs_on_key", unique: true
   end
 
+  create_table "admin_users", force: :cascade do |t|
+    t.string "email", default: "", null: false
+    t.string "encrypted_password", default: "", null: false
+    t.string "reset_password_token"
+    t.datetime "reset_password_sent_at"
+    t.datetime "remember_created_at"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["email"], name: "index_admin_users_on_email", unique: true
+    t.index ["reset_password_token"], name: "index_admin_users_on_reset_password_token", unique: true
+  end
+
+  create_table "conversation_users", force: :cascade do |t|
```

Comment:
Coversations should not be here, right?

Reply:
I think it might be just changing the table order in schema?

Reply:
Maybe the comment before was because of the the other PR that was already merged. Seems that it's okay now 👍